src/services/jwt.service.js

```diff
@@ -0,0 +1,36 @@
+const jwt = require('jsonwebtoken');
+
+const sign = (user) => {
+  const token = jwt.sign(user, process.env.JWT_KEY, { expiresIn: '30m' });
+
+  return token;
+};
+
+const verify = (token) => {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (e) {
+    return null;
```

Comment:
Returning null on verification failure is a valid approach, but consider logging the error for debugging purposes. This can help identify issues with token verification.